### DISCUSSION TITLE
How to get in-flight requests count with Pool

### DISCUSSION THREAD

**Discussion by StefanCenusa** [nodejs/undici#1160] [Q&A] [answered]
Hello, folks!

I am looking to replace the nodejs native http client with undici and I need to **track the in-flight requests** when using undici's `Pool`. I've searched all over github and docs, but can't find a nice way to do that. Size and queued requests on a Pool are on `Symbols` which are not exported. How is it recommended to get that info?

Thanks!

**Comment by mcollina** [comment, marked as answer]
I think the best way is to send a PR to expose some of that data!
